ros/cpp/src/ptg_datahub/src/ptg_datahub.cpp

```diff
@@ -81,7 +89,7 @@ class MinimalPublisher : public rclcpp::Node
     std::thread StartTCPServerThread(int port)
```

Comment:
Maybe not a "this branch" thing, but I think something better could be used here as an identifier than the port integer. E.g. we could make an enum of the different sensor stream "types" and take in a value from that enum here. Then we have some center to index into various mappings (the enum values) and have a natural constraint in the values that can be provided to server threads: enum values (e.g. don't hvae to change this port-range check when expanding/changing sensor port attribution).